### PR TITLE
[12.0][MIG] stock_account_quantity_history_location

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
Standard migration

Depends on
- [x]  #65 
- [x]  #74 

@Eficent

### CODE REVIEW

**Comment by LoisRForgeFlow** [comment]
@hveficent could you rebase? dependency is met now.

**Comment by rousseldenis** [comment]
@hveficent Please see runbot warning:

`2019-08-26 13:24:27,612 161 WARNING openerp_test odoo.tools.view_validation: Invalid XML: An alert (class alert-*) must have an alert, alertdialog or status role. Please use alert and alertdialog only for what expects to stop any activity to be read immediatly.
2019-08-26 13:24:27,613 161 WARNING openerp_test odoo.addons.base.models.ir_ui_view: Invalid view Valuation Report definition in stock_account_quantity_history_location/wizards/stock_quantity_history.xml
<?xml version="1.0"?>
<field name="compute_at_date" position="before">
<field name="location_id"/>
<field name="include_child_locations" attrs="{'invisible': [('location_id', '=', False)]}"/>
<div class="alert alert-warning" colspan="2" attrs="{'invisible': [('location_id', '=', False)]}" style="margin-bottom:0px;">
<p><i class="fa fa-info-circle"/> The accounting valuation is not calculated when you specify a location.
</p>
</div>
</field>
`

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM

**Comment by HviorForgeFlow** [comment]
@ageficent @aheficent  @rousseldenis could you review this please? Thanks!

File: stock_account_quantity_history_location/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+from . import wizards
```
Comment (rousseldenis) [inline stock_account_quantity_history_location/__init__.py:1]: Missing new line

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review. Few comment.

Do you plan to add test for new function added ?

**Review by rousseldenis — APPROVED** [review]

**Comment by HviorForgeFlow** [comment]
@rousseldenis I'm not planning to add test right now...

**Comment by rousseldenis** [comment]
Ok, let's merge

**Pull request merged**